aten/src/ATen/native/cuda/RowwiseScaledMM.cu

```diff
@@ -3,9 +3,15 @@
 #include <ATen/core/Tensor.h>
 #include <ATen/cuda/CUDAContext.h>
 #include <ATen/cuda/nvrtc_stub/ATenNVRTC.h>
+#include <c10/macros/Macros.h>
+
+// Two warninngs in Cutlass included header files
+C10_DIAGNOSTIC_PUSH_AND_IGNORED_IF_DEFINED("-Wset-but-not-used")
```

Comment:
Technically it's good to pair each `C10_DIAGNOSTIC_PUSH` with `C10_DIAGNOSTIC_POP`...